src/Corax/IndexFieldsMapping.cs

```diff
@@ -153,7 +186,7 @@ public string FieldNameAsString
     {
         get
         {
-            return _fieldName ?? (_fieldName = FieldName.ToString());
+            return _fieldName ??= FieldName.ToString();
```

Comment:
Invariant culture maybe?
@ppekrol

Reply:
Why? That is a string value, not a number, no?